packages/form-layout/test/dom/__snapshots__/form-layout.test.snap.js

```diff
@@ -1,7 +1,7 @@
 /* @web/test-runner snapshot v1 */
 export const snapshots = {};
 
-snapshots["vaadin-form-layout auto-responsive basic host default"] = 
+snapshots["vaadin-form-layout auto-responsive basic host default"] =
```

Comment:
I think these whitespace changes should not be needed. Please consider removing `form-layout.test.snap.js` file and running `yarn update:snapshots --group form-layout` again, then the file will be regenerated correctly.

This would rearrange some snapshots to match the order of the newly added tests.

Reply:
Thanks, done.